apps/admin-gui/src/app/shared/components/members-candidates-list/members-candidates-list.component.html

```diff
@@ -99,6 +108,6 @@
   </perun-web-apps-table-wrapper>
 </div>
 
-<perun-web-apps-alert *ngIf="members.length === 0" alert_type="warn">
+<app-alert *ngIf="members.length === 0" alert_type="warn">
```

Comment:
This is the correct one. Now with new eslint rules we want to use `perun-web-apps` pre-selector for alers.
```suggestion
<perun-web-apps-alert *ngIf="members.length === 0" alert_type="warn">
```